affordance_util/include/affordance_util/affordance_util.hpp

```diff
@@ -209,7 +215,38 @@ CcModel compose_cc_model_slist(const RobotDescription &robot_description, const
  */
 Eigen::MatrixXd compose_cc_model_slist(const RobotDescription &robot_description, const ScrewInfo &aff_info,
                                        const VirtualScrewOrder &vir_screw_order = VirtualScrewOrder::XYZ);
-
+/** 
+ * @brief Given a urdf pose transform it to a rotation and translation matrix.
```

Comment:
Given a urdf-type pose converts it to a homogeneous transformation matrix of Eigen::Matrix4d type.